__test__/lib/array_binary_search.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
The tests in this file do not match the lab - looks like they are whiteboarding tests!